content/applications/hr/time_off.rst

```diff
@@ -237,6 +237,23 @@ Enter the following information on the form:
 - :guilabel:`Based on worked time`: Enable this option if time off accrual is determined by the
   employee's worked hours. Days **not** considered as worked time do **not** contribute to the
   accrual plan in Odoo.
+
+  .. example::
+     An employee is granted time off from an accrual plan configured to accrue one day of vacation
+     for every five days worked. The accrual plan is based on the employee's worked time (the
+     :guilabel:`Based on worked time` checkbox is ticked).
+
+     The employee works standard 40 hour weeks. According to the accrual plan, they should earn
```

Comment:
hyphen here since it's qualifying a noun

```suggestion
     The employee works standard 40-hour weeks. According to the accrual plan, they should earn
```